website/docs/license.md

```diff
@@ -6,8 +6,7 @@ sidebar_position: 8
 
 Copyright (C) 2017-2023 [The ORT Project Authors](https://github.com/oss-review-toolkit/ort/blob/main/NOTICE).
 
-See the [LICENSE](https://github.com/oss-review-toolkit/ort/blob/main/LICENSE) file in the root of this project for
-license details.
+OSS Review Toolkit (ORT) is licensed under the [Apache License, Version 2.0](https://github.com/oss-review-toolkit/ort/blob/main/LICENSE).
```

Comment:
There's now an ugly duplication of "OSS Review Toolkit (ORT) is ..." here and below. I wonder whether we should combine this further to

```
OSS Review Toolkit (ORT) is a [Linux Foundation project]https://www.linuxfoundation.org) as part of [ACT](https://automatecompliance.org/) and licensed under the [Apache License, Version 2.0](https://github.com/oss-review-toolkit/ort/blob/main/LICENSE).
```

@mnonnenmacher what do you think?

Reply:
I did exactly like that but then decided just commit first one, to see if was not my impression only.
Changing to that

Reply:
How about this way:
![image](https://github.com/oss-review-toolkit/ort/assets/418176/4a9ee90f-5159-43a7-a033-96a758c49df8)